src/ripple/protocol/BuildInfo.h

```diff
@@ -68,6 +68,18 @@ getFullVersionString();
 std::uint64_t
 getEncodedVersion();
 
+/** Compare the local node's rippled server software version with the given one.
+
+    @note We only compare the major version, minor version, and patch version.
+    @param othersVersion another node's rippled server software version
+    @return true if local node's rippled server software version is lower,
+            false otherwise.
+*/
+bool
+localVersionLow(std::uint64_t othersVersion);
+
+static constexpr float versionUpgradeWarningThreshold = 0.6;
```

Comment:
This does not belong here.

Reply:
Make sense. It is deleted. It is changed to `60/100` and place in the code that check the versions:
https://github.com/pwang200/rippled/blob/7eba307c5c0d8e81e3e5eab195cf5227734bc9ec/src/ripple/app/ledger/impl/LedgerMaster.cpp#L1074-L1076